packages/expo-cli/src/commands/start/TerminalUI.ts

```diff
@@ -56,23 +57,24 @@ const printUsageAsync = async (
   projectRoot: string,
   options: Pick<
     StartOptions,
-    'webOnly' | 'devClient' | 'isWebSocketsEnabled' | 'isRemoteReloadingEnabled'
+    'webOnly' | 'devClient' | 'isWebSocketsEnabled' | 'isRemoteReloadingEnabled' | 'platforms'
   > = {}
 ) => {
   const { dev } = await ProjectSettings.readAsync(projectRoot);
   const openDevToolsAtStartup = await shouldOpenDevToolsOnStartupAsync();
   const devMode = dev ? 'development' : 'production';
   const currentToggle = openDevToolsAtStartup ? 'enabled' : 'disabled';
+  const { platforms = ['android', 'ios', 'web'] } = options;
 
   const isMac = process.platform === 'darwin';
 
   logCommandsTable([
     [],
-    ['a', `open Android`],
-    ['shift+a', `select a device or emulator`],
-    isMac && ['i', `open iOS simulator`],
-    isMac && ['shift+i', `select a simulator`],
-    ['w', `open web`],
+    platforms.includes('android') && ['a', `open Android`],
+    platforms.includes('android') && ['shift+a', `select a device or emulator`],
+    platforms.includes('ios') && isMac && ['i', `open iOS simulator`],
+    platforms.includes('ios') && isMac && ['shift+i', `select a simulator`],
+    platforms.includes('web') && ['w', `open web`],
```

Comment:
This would still be shown in projects that use Next.js

Reply:
not if you set `"platforms": ["ios", "android"]` in app config